src/if-else.c

```diff
@@ -52,8 +52,9 @@ r_obj* vec_if_else(
   struct r_lazy error_call
 ) {
   obj_check_vector(condition, VCTRS_ALLOW_NULL_no, p_condition_arg, error_call);
+  check_condition_index(condition, p_condition_arg, error_call);
+
   const r_ssize size = r_length(condition);
```

Comment:
Of note, not everyone needed the size check. `if_else()` takes the size from `condition`, so no check is required here. So that makes me feel even better about this.